browser/resources/settings/brave_wallet_page/wallet_networks_subpage.js

```diff
@@ -115,6 +152,23 @@ class SettingsWalletNetworksSubpage extends SettingsWalletNetworksSubpageBase {
         then(success => { this.updateNetworks() })
   }
 
+  onResetActionTapped_(event) {
```

Comment:
Do we have any browser or js tests to cover such code?

Reply:
Not that I know of yet, good to have for sure but I won't block this PR by it.